libs/ui-toolkit/src/components/theme-switcher/theme-switcher.tsx

```diff
@@ -13,7 +13,7 @@ export const ThemeSwitcher = ({
   sunClassName?: string;
   moonClassName?: string;
 }) => {
-  const sunClasses = classNames('dark:hidden text-black', sunClassName);
+  const sunClasses = classNames('dark:hidden text-white', sunClassName);
```

Comment:
Chore (blocking): This change would mean the icon is invisible on sites with white backgrounds in light mode. I think we need a prop to specify if the icon's always on a dark background.

Reply:
My bad not sure why I changed this its unrelated. reverting